website/index.html

```diff
@@ -413,6 +413,14 @@ <h2 id="quick-start">Quick Start</h2>
 
 sudo service clickhouse-server start
 clickhouse-client
+</pre>
+<pre id="instruction_rpm" style="display: none;">
+sudo yum install yum-utils
+sudo yum-config-manager --add-repo https://repo.yandex.ru/clickhouse/rpm/stable/x86_64
+sudo yum install clickhouse-server clickhouse-client
+
```

Comment:
You missed pub key import:
`$ rpm --import https://repo.yandex.ru/clickhouse/CLICKHOUSE-KEY.GPG`